### ISSUE TITLE
Feature request: Group nodes based on (configurable) labels

### ISSUE THREAD

**Issue by sfudeus** [hjacobs/kube-ops-view#250]
It would be nice if nodes could be grouped according to a configurable node-label. 
The idea behind this is to be able to visualize physical infrastructural topology like rack or router association. While a single group would already be helpful, multiple layers could be put to good use (e.g. dc, az/iz, room, router, rack).

**Comment by hjacobs** [comment]
Do you have a proposal for how to visualize this?

**Comment by f3d3Cz** [comment]
I completely agree with this. Having different groups of nodes inside the cluster group would probably be a good starting point.
